### PR TITLE
Wagtail 5 support

### PR DESCRIPTION
**Pull request by jams2**
- Adds support for Wagtail 5
- Drops Python 3.7 from the CI matrix as it's unsupported
- Drops Django 4.0 from the CI matrix as it's unsupported
- Renames master to main in tox.ini (in line with Wagtail upstream)

I have tested importing documents via:
- OneDrive import
- Google import
- Local document import

I have tested constructing StructBlocks using custom converter classes.

### CODE REVIEW

File: wagtail_content_import/admin_views.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 def confirm_dialog(request):
     return render_modal_workflow(
         request,
-        'wagtail_content_import/confirm_dialog.html',
-        None,
-        {},
+        html_template='wagtail_content_import/confirm_dialog.html',
+        js_template=None,
+        template_vars={},
```
Comment (jams2) [inline wagtail_content_import/admin_views.py:9]: Unrelated change, just improves readability.

**Review by zerolab — CHANGES_REQUESTED** [review]
Needs a couple of small changes. LGTM otherwise (caveat: haven't got a chance to test locally)

File: tox.ini [hunk 2]
```diff
@@ -36,15 +34,16 @@ deps =
     django3.2: Django>=3.2,<3.3
     django4.0: Django>=4.0,<4.1
     django4.1: Django>=4.1,<4.2
```
[the remark this answers is not in the document]
Comment (jams2) [reply tox.ini]: Fixed in https://github.com/torchbox/wagtail-content-import/pull/59/commits/92e62bd3d9f97e2e9fba463781954301a31b8418

File: tox.ini [hunk 3]
```diff
@@ -3,14 +3,12 @@ skipsdist = True
 usedevelop = True
 
 envlist =
-    python{3.7}-django{3.2}-wagtail{4.1,4.2,master}-{sqlite,postgres}
-    python{3.8,3.9,3.10}-django{3.2}-wagtail{4.1,4.2,master}-{sqlite,postgres}
-    python{3.8,3.9,3.10}-django{4.0,4.1}-wagtail{4.1,4.2,master}-{sqlite,postgres}
-    python{3.11}-django{4.1}-wagtail{4.1,4.2,master}-{sqlite,postgres}
+    python{3.8,3.9,3.10}-django3.2-wagtail{4.1,4.2,5.0,5.1,main}-{sqlite,postgres}
```
[the remark this answers is not in the document]
Comment (jams2) [reply tox.ini]: Thanks, removed it in https://github.com/torchbox/wagtail-content-import/pull/59/commits/92e62bd3d9f97e2e9fba463781954301a31b8418 - this seems to be the simplest I can get it, due to:
- Django 3.2 not supporting Python 3.11; and
- Wagtail 4.1 not support Django 4.2.

[the remark this answers is not in the document]
Comment (jams2) [reply tox.ini]: Thanks, reshuffled this a bit in https://github.com/torchbox/wagtail-content-import/pull/59/commits/f45bfef78da467c67ffe7d39fce60ce6a24ba08f, it's a little easier on the eyes/brain.

File: tox.ini (hunk 2 above)
[the remark this answers is not in the document]
Comment (gasman) [reply tox.ini]: Nitpick - the commit message says 4.1 (which _isn't_ EOL yet as it's an LTS release), although it is indeed removing 4.2.

**Review by gasman — APPROVED** [review]
Other than the commit message nitpick, LGTM!

File: tox.ini (hunk 2 above)
[the remark this answers is not in the document]
Comment (jams2) [reply tox.ini]: Thanks, reworded.

**Pull request merged**